### PR TITLE
docs: Add some details about coordinates

### PR DESCRIPTION
**Pull request by matthiasclasen**
Mention what the effective values are when no coordinates have been set.

### CODE REVIEW

File: src/hb-font.cc [hunk 1]
```diff
@@ -2236,10 +2236,10 @@ hb_font_get_var_coords_normalized (hb_font_t    *font,
  *
  * Return value: coordinates array
  *
- * Since: REPLACEME
+ * Since: 3.3.0
```
Comment (behdad) [inline src/hb-font.cc:2247]: This gets filled in at release time based on the chosen version.

**Comment by behdad** [comment]
May I suggest an improvement? It's possible that the caller of the set_design_coords / set_normalized_coords sets partial coords. In that case, any unset coords get the 0 / default coords. Can you rewrite to say "any unset coords get 0 / default value"?

File: src/hb-font.cc (hunk 1 above)
Comment (matthiasclasen) [reply src/hb-font.cc:2247]: oh, nice. I should copy that trick

**Comment by matthiasclasen** [comment]
Still not quite right. The function just returns what has been set by the previous set_var_coords call, which may be a partial array, and must be interpreted as zero-padded in the normalized case and as default-value-padded in the design case.

**Pull request merged**